src/client/eppo-precomputed-client.ts

```diff
@@ -57,12 +57,18 @@ export default class EppoPrecomputedClient {
     private isObfuscated = false,
   ) {}
 
+  public setPrecomputedFlagsRequestParameters(
```

Comment:
Is used here https://github.com/Eppo-exp/js-client-sdk/blob/d584a9e1bed0f2da756121bd0b84044c174b9909/src/index.ts#L598, so I shouldn't have deleted it